services/web/client/source/resource/qxapp/project-v0.0.1.json

```diff
@@ -5,19 +5,19 @@
   "type": "object",
   "additionalProperties": false,
```

Comment:
this file is duplicated in apihub. -> needs to be updated in apihub or the webserver will not be in sync
we need a mechanism to prevent this duplication: the webserver should provide the jsonschema to the webclient

Reply:
Deleted and case created #485